terraform/ecs.tf

```diff
@@ -30,11 +38,12 @@ resource "aws_ecs_task_definition" "webapp" {
 }
 
 resource "aws_ecs_service" "webapp" {
-  name            = "beacons-webapp"
-  cluster         = aws_ecs_cluster.main.id
-  task_definition = aws_ecs_task_definition.webapp.arn
-  desired_count   = var.webapp_count
-  launch_type     = "FARGATE"
+  name             = "${var.env}-beacons-webapp"
+  cluster          = aws_ecs_cluster.main.id
+  task_definition  = aws_ecs_task_definition.webapp.arn
+  desired_count    = var.webapp_count
+  launch_type      = "FARGATE"
+  platform_version = "1.3.0"
```

Comment:
Specifying version, because otherwise AWS will update Fargate and break our VPC endpoints 😭